Sources/MapboxMaps/Gestures/GestureOptions.swift

```diff
@@ -1,100 +1,52 @@
-        by dragging or swiping with one finger.
-
-        This property controls only user interactions with the map. If you set the
-        value of this property to `false`, you may still change the map location
-        programmatically.
-
-        The default value of this property is `true`.
-    */
-    public var scrollEnabled: Bool = true
-
-    /**
-        The scrolling mode the user is allowed to use to interact with the map.
-
-        `horizontal` only allows the user to scroll horizontally on the map,
-        restricting a user's ability to scroll vertically.
-
-        `vertical` only allows the user to scroll vertically on the map,
-        restricting a user's ability to scroll horizontally.
-
-        `horizontalAndVertical` allows the user to scroll both horizontally and vertically
-        on the map.
-
-        By default, this property is set to `MGLPanScrollingModeDefault`.
-     */
-    public var scrollingMode: PanScrollingMode = .horizontalAndVertical
+    /// Whether the pitch gesture is enabled. Defaults to `true`.
+    public var pitchEnabled: Bool = true
 
-    /**
-        A Boolean value that determines whether the user may change the pitch (tilt) of
-        the map.
+    /// Whether double tapping the map with one touch results in a zoom-in animation.
+    /// Defaults to `true`.
+    public var doubleTapToZoomInEnabled: Bool = true
 
-        When this property is set to `true`, the default, the user may tilt the map by
-        vertically dragging two fingers.
+    /// Whether single tapping the map with two touches results in a zoom-out animation.
+    /// Defaults to `true`.
+    public var doubleTouchToZoomOutEnabled: Bool = true
 
-        This property controls only user interactions with the map. If you set the
-        value of this property to `false`, you may still change the pitch of the map
-        programmatically.
+    /// Whether the quick zoom gesture is enabled. Defaults to `true`.
+    public var quickZoomEnabled: Bool = true
 
-        The default value of this property is `true`.
-    */
-    public var pitchEnabled: Bool = true
+    /// Configures the directions in which the map is allowed to move during a pan gesture.
+    /// Defaults to `PanMode.horizontalAndVertical`. Called `scrollMode` in
+    /// the Android SDK for consistency with platform conventions.
+    public var panMode: PanMode = .horizontalAndVertical
 
-    /**
-        A floating-point value that determines the rate of deceleration after the user lifts their finger.
-    */
-    public var decelerationRate: CGFloat = UIScrollView.DecelerationRate.normal.rawValue
+    /// A constant factor that determines how quickly pan deceleration animations happen.
+    /// Multiplied with the velocity vector once per millisecond during deceleration animations.
+    /// Defaults to `UIScrollView.DecelerationRate.normal.rawValue`
+    public var panDecelerationFactor: CGFloat = UIScrollView.DecelerationRate.normal.rawValue
```

Comment:
everywhere else in the code, this has been called decelerationFactor. Why is it that the options have a prepended pan in front of it?

Reply:
We want to indicate that this setting only applies to pan. In the other locations, the meaning is clear from the context. Later, we might add other separate deceleration factor options for other gestures, so the prefix sets us up to be able to do that in an additive way.